api/src/main/java/io/strimzi/api/kafka/model/KafkaBridgeSpec.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.fabric8.kubernetes.api.model.ResourceRequirements;
+import io.strimzi.api.kafka.model.template.KafkaBridgeTemplate;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.strimzi.crdgenerator.annotations.Minimum;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "replicas", "bootstrapServers", "authentication",
+        "tls", "http", "image", "resources", "jvmOptions",
+        "logging", "metrics", "template"})
+@EqualsAndHashCode
+public class KafkaBridgeSpec implements UnknownPropertyPreserving, Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private int replicas;
+    private String bootstrapServers;
+
+    private KafkaBridgeAuthentication authentication;
```

Comment:
No I don't think so. I would avoid to have this here for now. The same for the following ones.

Reply:
Are we ok with just commenting it out, or you want me to remove it from the code completely?

Reply:
I am for removing, not heaving "zombie" code.

Reply:
I think we should implement it. Without auth and TLS it will be completely useless. Nobody will be able to use it beyond 5 minutes playing in the dev environment. :-(

Also, removing it from the code will mean a lot of work now as well as later to get it back. Maybe we should just add to the description big bold **Currently not supported!**? That would be feasible for me.

Reply:
> I think we should implement it
If you can get me more time, of course.

Reply:
Commented as **Currently not supported**